mining/src/mempool/model/frontier.rs

```diff
@@ -394,13 +410,13 @@ mod tests {
             let args = FeerateEstimatorArgs { network_blocks_per_second: 1, maximum_mass_per_block: 500_000 };
             // We are testing that the build function actually returns and is not looping indefinitely
             let estimator = frontier.build_feerate_estimator(args);
-            let estimations = estimator.calc_estimations(1.0);
+            let estimations = estimator.calc_estimations(MIN_FEERATE);
 
             let buckets = estimations.ordered_buckets();
             // Test for the absence of NaN, infinite or zero values in buckets
             for b in buckets.iter() {
                 assert!(
-                    b.feerate.is_normal() && b.feerate >= 1.0,
+                    b.feerate.is_normal() && b.feerate >= MIN_FEERATE - EPS,
```

Comment:
Why is the epsilon here necessary?

Reply:
I've seen it fail in CI tests (very rarely, but it happened) due to platform specific floating op precision